packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1988,10 +1981,14 @@ export default class Meeting extends StatelessWebexPlugin {
         this.selfId === contentShare.beneficiaryId &&
         contentShare.disposition === FLOOR_ACTION.GRANTED
       ) {
-        if (this.mediaProperties.shareTrack?.readyState === 'ended') {
+        // @ts-ignore originalTrack is private - this will be fixed when SPARK-399694 are SPARK-399695 done
+        const localShareTrack = this.mediaProperties.shareTrack?.originalTrack;
```

Comment:
why do we need the original track here , doesnt publishTrack takes in the custom track

Reply:
we're calling unpublishTrack() and for now that still takes raw tracks and it compares them with originalTrack of the webrtc-core tracks that we have on meeting.mediaProperties, so we need to use originalTrack, otherwise it won't match and unpublishTracks() will not know which track we're trying to unpublish